jplephem/spk.py

```diff
@@ -65,6 +74,12 @@ def comments(self):
         """Return the file comments, as a string."""
         return self.daf.comments()
 
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.close()
+
+    def __enter__(self):
```

Comment:
A tiny nit: could "enter" go before "exit"? That's the order I'm used to reading them in.